camel/models/sglang_model.py

```diff
@@ -97,9 +97,17 @@ def __init__(
     def _start_server(self) -> None:
         try:
             if not self._url:
+                tool_call_flag = self.model_config_dict.get('tools')
```

Comment:
nit (consistency in string quotes):
```suggestion
                tool_call_flag = self.model_config_dict.get("tools")
```